presto-testing/src/main/java/io/prestosql/testing/AbstractTestAggregations.java

```diff
@@ -718,8 +718,8 @@ public void testApproximateCountDistinct()
         assertQuery("SELECT approx_distinct(orderdate, 0.023) FROM orders", "SELECT 2443");
 
         // test timestamp
-        assertQuery("SELECT approx_distinct(CAST(orderdate AS TIMESTAMP)) FROM orders", "SELECT 2384");
-        assertQuery("SELECT approx_distinct(CAST(orderdate AS TIMESTAMP), 0.023) FROM orders", "SELECT 2384");
+        assertQuery("SELECT approx_distinct(CAST(orderdate AS TIMESTAMP)) FROM orders", "SELECT 2379");
```

Comment:
We should document that `checksum()` changed for timestamp in this release. (we don't guarantee that, but good to tell people when we are aware)

Reply:
This is actually a different issue. xxhash for a value with p <= 3 now returns a different value because, so approx_distinct produces different results.